src/FakeItEasy/Core/BootstrapperLocator.cs

```diff
@@ -34,7 +34,7 @@ public static IBootstrapper FindBootstrapper()
 
             var bootstrapperType = candidateTypes.FirstOrDefault() ?? typeof(DefaultBootstrapper);
 
-            return (IBootstrapper)Activator.CreateInstance(bootstrapperType);
+            return (IBootstrapper)Activator.CreateInstance(bootstrapperType)!;

```

Comment:
Activator.CreateInstance can only return null if the type argument is `Nullable<T>`.